tests/test_mturk.py

```diff
@@ -172,8 +172,45 @@ def test_hits_only(hit):
             pass
 
 
+@pytest.fixture
+def worker_id():
+    # Get a worker ID from the environment or tests/config.py
+    import os
+    workerid = os.getenv('mturk_worker_id')
+    if not workerid:
+        try:
+            from . import config
+            workerid = config.mturk_worker_id
+        except Exception:
+            pass
+    if not workerid:
+        raise FixtureConfigurationError(
+            'No "mturk_worker_id" value found. '
+            'Either set this value or skip these tests with '
+            '`pytest -m "not mturkworker"`'
+        )
+    return workerid
+
+
+class MTurkTestBase(object):
+
+    def _make_qtype(self, mturk, name=None):
+        import socket
+        hostname = socket.gethostname()
+        if name is None:
+            name = "{}:{}".format(hostname, generate_random_id(size=32))
+
+        qtype = mturk.create_qualification_type(
+            name=name,
+            description=TEST_QUALIFICATION_DESCRIPTION,
+            status='Active',
+        )
+        self._qtypes_to_purge.append(qtype['id'])
+        return qtype
```

Comment:
It's tied into the `with_cleanup` fixture, but I'll see if I can extract it into a separate one.